cmd/hatchet-migrate/migrate/migrations/20250422170746_v1_0_16.sql

```diff
@@ -0,0 +1,11 @@
+-- +goose Up
+-- +goose StatementBegin
+ALTER TABLE v1_tasks_olap
+ADD COLUMN step_readable_id TEXT NOT NULL DEFAULT '';
```

Comment:
important: I was thinking making this non-null would give us improved behavior in the code + it should be non-null going forward, but happy to make this nullable if that doesn't make sense!